src/EscapeAnalysis.jl

```diff
@@ -267,7 +267,7 @@ function find_escapes(ir::IRCode, nargs::Int)
                     # TODO: we can apply similar strategy like builtin calls
                     #       to specialize some foreigncalls
                     foreigncall_nargs = length((stmt.args[3])::SimpleVector)
-                    add_change!(stmt.args[1], ir, Escape(), changes)
+                    push!(changes, (stmt.args[1], Escape()))
```

Comment:
This seems good for now. It almost never escapes (only on TypeError), so eventually we probably want to model it, but right now it is rather overcomplicated (and apparently undocumented?) in the representation and runtime.